pkg/virt-config/virt-config.go

```diff
@@ -136,6 +141,21 @@ func (c *ClusterConfig) GetCpuModelToFeaturesToDisableMap() map[string]*cmdv1.Fe
 		"Skylake-Server-noTSX-IBRS": {Features: []string{"mpx"}},
 		"Skylake-Server":            {Features: []string{"mpx"}},
 	}
+
+	cpuModelToFeaturesToDisableCM, err := cpuModelConfigMapLister.Get(util.CpuModelDisableFeaturesConfigMapName)
+	if err != nil {
+		return cpuModelToFeaturesToDisable
+	}
+
+	for model, featuresString := range cpuModelToFeaturesToDisableCM.Data {
+		features := strings.Split(featuresString, ",")
```

Comment:
It's always hard work with a string list. We should sanitize the list somewhere in the code, maybe in the `mergeStringSlices` function.
Since this slice is manually created there should be some typo, as a random number of spaces. e.g. This `"aes,    mpx"` will fail since `     mpx` is not trimmed.

Reply:
Done